sycl/source/detail/memory_manager.cpp

```diff
@@ -225,19 +240,31 @@ void copyD2H(SYCLMemObjT *SYCLMemObj, cl_mem SrcMem, QueueImplPtr SrcQueue,
   if (1 == DimDst && 1 == DimSrc) {
     CHECK_OCL_CODE(clEnqueueReadBuffer(
         CLQueue, SrcMem,
-        /*blocking_read=*/CL_FALSE, DstOffset[0], DstAccessRange[0],
-        DstMem + DstOffset[0], DepEvents.size(), &DepEvents[0], &OutEvent));
+	/*blocking_read=*/CL_FALSE, DstOffset[0], DstAccessRange[0],
+	DstMem + DstOffset[0], DepEvents.size(), &DepEvents[0], &OutEvent));
   } else {
-    size_t BufferRowPitch = (1 == DimSrc) ? 0 : SrcSize[0];
-    size_t BufferSlicePitch = (3 == DimSrc) ? SrcSize[0] * SrcSize[1] : 0;
 
```

Comment:
Can probably remove this white space as well to sit the new "if (...)" below the else